docs/mmds.md

```diff
@@ -66,12 +65,12 @@ use cases will encompass the following sequence of steps:
    place before the microVM starts running, but may also happen at a later
    time. Guest MMDS requests which arrive prior to contents being available
    receive a *NotFound* response.
-1. The contents are saved to the data store.
-1. The guest sends a `GET` request for the secret key, which is intercepted by
+2. The contents are saved to the data store.
```

Comment:
nit: Markdown supports numbered lists in both formats:
```
1.
1.
1. 
```
and
```
1.
2.
3.
```

And most (if not all) our documents use the first option, I suggest you discard this change to keep things consistent.

Reply:
Sorry, I my editor did it by itself 😅, I fixed it now